content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,281 @@ Feature: Is it Friday yet?
 
 Congratulations! You've got your first green Cucumber scenario.
 
+# Evaluate a scenario
+Now that we've forced a scenario to pass, let's write one that passes or fails based on logic. Let's update our statement to evaluate whether or not `today` is equal to `"Friday"`.
+
+{{% block "javascript" %}}
+```javascript
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```ruby
+def is_it_friday?(day)
+  answer = (day == 'Friday') ? 'TGIF' : 'Nope'
+end
+```
+{{% /block %}}
+
+Run Cucumber again:
+
+{{% block "javascript" %}}
+```shell
+...
+1 scenario (1 passed)
+3 steps (3 passed)
+0m00.001s
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```shell
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario: Sunday isn't Friday        # features/is_it_friday_yet.feature:4
+    Given today is Sunday              # features/step_definitions/stepdefs.rb:8
+    When I ask whether it's Friday yet # features/step_definitions/stepdefs.rb:12
+    Then I should be told "Nope"       # features/step_definitions/stepdefs.rb:16
+
+1 scenario (1 passed)
+3 steps (3 passed)
+0m0.049s
+```
+{{% /block %}}
+
+# Multiple scenarios
+We've seen what happens when it *isn't* Friday, but what about when it *is* Friday? Fortunately, we've already written the statement to evaluate whether or not `today` is equal to `"Friday"` and return `"TGIF"` if it is. So, let's write a scenario where today is Friday!
```

Comment:
When switching the order, the text here will have to change. Add the scenario; see that it fails, add the logic.